test_scripts/API/CancelInteraction/004_SendCancelInteraction_Slider_HMIReceivesRequest.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--  1) app1 is registered on SDL
+--
+--  Steps:
+--  1) app1 sends a sends an Slider RPC
+--  2) app1 sends a CancelInteraction Request with the functionID of Slider
+--  3) the HMI receives the CancelInteraction Request and replies
+--
+--  Expected:
+--  1) app1 receives SUCCESS from the CancelInteraction
+--  2) app1 receives ABORTED from the Slider
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local variables ]]
+local rpcInteraction = {
+  name = "Slider",
+  hmi_name = "UI.Slider",
+  params = {
+    numTicks = 16,
```

Comment:
Add `cancelID` to this request and `CancelInteraction`

Reply:
added to both in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2206/commits/471e8b5b7364595f4a5d0d0cb371f70f2d31a730